docs/cities/manchester/roles-and-duties.md

```diff
@@ -0,0 +1,111 @@
+---
+id: roles-and-duties
+title: Roles & Duties
```

Comment:
Maybe: Roles and Duties for Saturday Classes. 
- volunteers may have roles and duties outside of this list that are more related to to general team tasks. The below roles are about Saturdays

Reply:
Hi Hirra, the initiative text has been reworked, could you please give a review again.
As for the title, I think I'll leave that as it is.